crosswordDSL/QuickSlow/grid1.yaml

```diff
@@ -0,0 +1,36 @@
+name: QuickSlow grid1 2018/03/01
+author: QuickSlow
```

Comment:
Author name doesn't show as per the example text file

Reply:
These are overwritten by the specific crossword. 
When a new crossword comes in, the appropriate grid is identified, and just the coords are copied across into the crossword obj.